src/lib/model/OutboundTransfersModel.js

```diff
@@ -625,13 +627,13 @@ class OutboundTransfersModel {
             expiration: this._getExpirationTimestamp()
         };
 
-        if(this._useQuoteSourceFSPAsTransferPayeeFSP) {
+        if (this._useQuoteSourceFSPAsTransferPayeeFSP) {
```

Comment:
please dont change whitespace unless there is a very good reason to do so.

Reply:
Deleted